iteration/iteration_test.go

```diff
@@ -119,7 +117,7 @@ func (s *TestIterationRepository) TestCreateChildIteration() {
 		StartAt:    &start,
 		EndAt:      &end,
 		Path:       parentPath,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c